src/main/java/ma/hibernate/util/HibernateUtil.java

```diff
@@ -0,0 +1,23 @@
+package ma.hibernate.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
```

Comment:
Are you need them, you have abstract class, they have all that you need

Reply:
same